bin.src/condorLogIngest.py

```diff
@@ -99,6 +99,7 @@ def run():
                 print("warning: %s does not exist." % filename)
             continue
         logIngestor.ingest(filename)
+    dbm.close()
```

Comment:
True, but it doesn't do any harm.   Eventually this code will transition to a method call in a new object that Orchestration will call, and I wanted to remind myself to close that connection explicitly, rather than waiting for it to be garbage collected.   Here it doesn't matter.